LogMonitor/src/LogMonitor/EventMonitor.cpp

```diff
@@ -710,3 +711,43 @@ EventMonitor::EnableEventLogChannel(
         EvtClose(channelConfig);
     }
 }
+
+/// <summary>
+/// XML Formatted Event
+/// </summary>
+std::wstring EventMonitor::XMLFormattedEvent()
+{
+    auto logFmt = L"<Source>%s</Source><Time>%s</Time><LogEntry><Channel>%s</Channel><Level>%s</Level><EventId>%u</EventId><Message>%s</Message></LogEntry>";

```

Comment:
Will be nice to have an engulfing tag for the whole log, I don't know what will be appropriate name here; but also match with the JSON schema (so, might need to re-arrange the location for `LogEntry`:
```xml
<Tag(need name)>
  <Source>....</Source>
  <LogEntry>
    <Time>...</Time>
    <Channel>...</Channel>
    <Level>...</Level>
     ....
  </LogEntry>
</Tag>
```

Reply:
Thank you for pointing this out. LogEntry tag re-arranged to engulf the entire log entry.
Sample XML output after change:
`<LogEntry><Source>EventLog</Source><Time>2023-01-16T05:53:48.000Z</Time><Channel>System</Channel><Level>Information</Level><EventId>12</EventId><Message>Process C:\Windows\System32\WUDFHost.exe (process ID:1772) reset policy scheme from {381b4222-f694-41f0-9685-ff5bb260df2e} to {381b4222-f694-41f0-9685-ff5bb260df2e}</Message></LogEntry>`

Sample JSON output after change:
```
{
  "LogEntry": {
    "Source": "EventLog",
    "Time": "2023-01-13T18:22:52.000Z",
    "Channel": "System",
    "Level": "Warning",
    "EventId": 700,
    "Message": "Power Manager has requested suppression of all input (INPUT_SUPPRESS_REQUEST=1)"
  }
}
```

Reply:
Was recommending that it matches with what we have in the ETW schema, same to all the other log sources:
```json
{
    "Source": "ETW",
    "LogEntry": {
        "Time": "2022-12-06T13:40:28.000Z",
        "ProviderId": "{DAA6A96B-F3E7-4D4D-A0D6-31A350E6A445}",
        "ProviderName": "Microsoft-Windows-WLAN-Driver",
        "DecodingSource": "DecodingSourceXMLFile",
        "Execution": {
            "ProcessId": 13804,
            "ThreadId": 13900
        },
        "Level": "Information",
        "Keyword": "0x8000000000000001",
        "EventId": 0,
        "EventData": {
            "FrameUniqueID": 5434658,
            "PortNumber": 0,
            "TID": 0,
            "PeerID": 0,
            "PayloadLength": 1286,
            "QueueLength": 0,
            "QueueState": "false",
            "CustomData1": 24,
            "CustomData2": 0,
            "CustomData3": 0
        }
    },
    "SchemaVersion": "1.0.0"
}
```
ref: https://github.com/microsoft/windows-container-tools/discussions/103

Reply:
I propose leaving the xml as is and putting out a discussion/proposal for engagement on a new XML schema and building that as it's own feature. Thoughts?

Reply:
XML and JSON Changes reverted.